### PR TITLE
Enhance studio runner to support running list of specific recording i…

### PR DESCRIPTION
**Pull request by eitany**
…ds belonging to multiple categories

### CODE REVIEW

File: playback/tape_cassettes/asynchronous/async_record_only_tape_cassette.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ def get_recording(self, recording_id):
     def iter_recording_ids(self, category, start_date=None, end_date=None):
         raise Exception("AsyncTapeCassette should only be used for recording, not playback")
 
+    def extract_recording_category(self, recording_id):
+        raise Exception("AsyncTapeCassette should only be used for recording, not playback")
```
Comment (urishalit) [inline playback/tape_cassettes/asynchronous/async_record_only_tape_cassette.py:65]: Exception is too broad - here and everywhere - it is bad practice

File: playback/studio/studio.py [hunk 2]
```diff
@@ -55,8 +79,14 @@ def _play_category(self, category):
         except Exception as ex:
             _logger.info(u'Cannot tune equalizer for category {} - {}'.format(category, ex))
             return ex
-        equalizer = Equalizer(find_matching_playable_recordings(
-            self.tape_recorder, tuning.playback_function, category,
-            self.lookup_properties), tuning.result_extractor, tuning.comparator,
-            comparison_data_extractor=tuning.comparison_data_extractor)
+
+        if recording_ids:
+            playable_recordings = by_id_playable_recordings(
```
Comment (urishalit) [inline playback/studio/studio.py:84]: Function names should be verbs/actions

**Review by urishalit — APPROVED** [review]

File: playback/studio/studio.py (hunk 2 above)
Comment (eitany) [reply playback/studio/studio.py:84]: this is not part of this PR and it is already used by users of the library so it is a breaking change

File: playback/tape_cassettes/asynchronous/async_record_only_tape_cassette.py (hunk 1 above)
Comment (eitany) [reply playback/tape_cassettes/asynchronous/async_record_only_tape_cassette.py:65]: The best fitting error I could find is TypeError, replaced with it.
From python doc:

exception TypeError
Raised when an operation or function is applied to an object of inappropriate type. The associated value is a string giving details about the type mismatch.

This exception may be raised by user code to indicate that an attempted operation on an object is not supported, and is not meant to be. If an object is meant to support a given operation but has not yet provided an implementation, NotImplementedError is the proper exception to raise.

**Pull request closed without merging**